contracts/util/SettingsConsts.sol

```diff
@@ -12,6 +12,9 @@ pragma solidity 0.5.17;
 contract SettingsConsts {
     /** Constants */
 
+    // It represents 100% with 2 decimal places.
+    uint256 public constant TEN_THOUSAND = 10000;
```

Comment:
I think it would help to understand the constant if we change the constant name with the corresponding meaning instead of value. Something like HUNDRED_PERCENT or similar